moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -92,6 +92,49 @@ const filterComponentFullNames = (value) => {
   ));
 };
 
+/**
+ * Get the user hidden column settings from local storage, clear them of
+ * outdated columns, and supplement with remaining default columns
+ * @returns {Object} hidden column settings from local storage
+ */
+const getExistingHiddenColumns = () => {
+  const existingHiddenColumnSettings = JSON.parse(
```

Comment:
this naming convention makes sense! but I found myself replacing 'existing' with 'previous' as I read through the code because I found the juxtaposition between 'previous' and 'current' to be a little clearer than 'existing' and 'current' which sound very similar in my mind. I definitely understand how 'existing' is more accurate but my suggestion would be to replace it with 'previous' for a more standard comparison.

Reply:
I agree, and I'm making this change - thank you! 'Previous' sounds much better when I'm reading through this now. 🙏